src/modelbench/run.py

```diff
@@ -101,10 +103,13 @@ def benchmark(
     anonymize=None,
     parallel=False,
 ) -> None:
+    start_time = datetime.now(timezone.utc)
     suts = find_suts_for_sut_argument(sut)
-    benchmarks = [b() for b in BenchmarkDefinition.__subclasses__() if b.__name__ in benchmark]
-    benchmark_scores = score_benchmarks(benchmarks, suts, max_instances, debug, parallel)
+    benchmark = [b() for b in BenchmarkDefinition.__subclasses__() if b.__name__ == benchmark_name][0]
+    benchmark_scores = score_benchmarks([benchmark], suts, max_instances, debug, parallel)
     generate_content(benchmark_scores, output_dir, anonymize, view_embed, custom_branding)
+    json_path = output_dir / f"benchmark_record-{benchmark.uid}.json"
+    dump_json(json_path, start_time, benchmark, benchmark_scores)
```

Comment:
I am perplexed! How does this work without importing the method from `modelbench.record`?

Reply:
Hah! That's because it doesn't work. I must have messed up the imports at the last minute, and that is not tested. I'll fix it. Thanks!

Reply:
Fixed, and test added.